numpy/lib/function_base.py

```diff
@@ -3445,6 +3446,7 @@ def percentile(a, q, axis=None, out=None,
     mean
     median : equivalent to ``percentile(..., 50)``
     nanpercentile
+    quantile
```

Comment:
Could add a summary here such as a slightly better worded version of "like `percentile`, but taking 0-1 instead of 0-100"